gammapy/estimators/utils.py

```diff
@@ -807,27 +807,21 @@ def get_combined_significance_maps(estimator, datasets):
                 * "significance" : joint significance map.
                 * "df" : degree of freedom map (one norm per valid bin).
                 * "npred_excess" : summed excess map.
-                * "estimator_results" : dictionary containing the estimator results for each dataset.
-    """
-    from .map.excess import ExcessMapEstimator
-    from .map.ts import TSMapEstimator
+                * "estimator_results" : dictionary containing the flux maps computed for each dataset.
 
-    if not isinstance(estimator, (ExcessMapEstimator, TSMapEstimator)):
-        raise TypeError(
-            f"estimator type should be ExcessMapEstimator or TSMapEstimator), got {type(estimator)} instead."
-        )
+    See also
+    --------
+    `get_combined_significance_maps` : same method but computing the significance maps from estimators and datasets.
```

Comment:
Dont want to have the quotes either. Fix for all please

```suggestion
    get_combined_significance_maps : same method but computing the significance maps from estimators and datasets.
```